presto-docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -14,19 +14,48 @@ Synopsis
 Description
 -----------
 
-Returns approximated statistics for the named table or for the results of a (limited) query.
-
-Statistics are returned for the specified columns, along with a summary row.
-(column_name will be ``NULL`` for the summary row).
-
-=====================   ============
-Column                  Description
-=====================   ============
-column_name             The name of the column
-data_size               The total size in bytes of all of the values in the column
-distinct_values_count   The number of distinct values in the column
-nulls_fractions         The portion of the values in the column that are ``NULL``
-row_count               The number of rows (only returned for the summary row)
-low_value               The lowest value found in this column (only for some types)
-high_value              The highest value found in this column (only for some types)
-=====================   ============
+``SHOW STATS FOR table``: Show statistics for a specific table.
+It provides information about the number of rows, number of nulls, minimum and maximum values,
+number of distinct values, and average length for each column in the table.
+
+``SHOW STATS FOR ( query )``: Show statistics for the result of a specific query.
+It provides the same information as ``SHOW STATS FOR table``, but for the result set of the query instead
+of a single table.
+
+Any statistics that are not populated or unavailable on the data source are returned as ``NULL``.
+Any additional statistics collected on the data source other than those listed below are not included in the output
```

Comment:
Lines 26-28: Your change merged two sentences into a very big sentence that I think would be easier to read and understand if it was separated into shorter sentences, similar to what I suggested in the first review. 

Consider replacing lines 26-28 with something like the following: 

```suggestion
Any additional statistics collected on the data source other than those listed below are not included in the output
of the ``SHOW STATS`` command. The summary row does not correspond to any specific column so the ``column_name`` value for the summary row is ``NULL``.
```